default_gateway.go

```diff
@@ -0,0 +1,158 @@
+package libnetwork
+
+import (
+	"fmt"
+
+	"github.com/docker/libnetwork/netlabel"
+	"github.com/docker/libnetwork/options"
+	"github.com/docker/libnetwork/types"
+)
+
+const (
+	libnGWNetwork = "GW_bridge"
```

Comment:
I think its better to keep things simple. I think someone already opened an issue to use '.' in service name.. if we add a special character someone will want to use it :) I changed the name to docker_gw_bridge to make it more docker specific and avoid the likelihood of conflict.